### ISSUE TITLE
Content - update current year income exceptions & additional transportation

### ISSUE THREAD

**Issue by Joshua-Lakusta** [bcgov/SIMS#4035]
As a student it is unclear exactly what type of income should be included for these exception questions

**Acceptance Criteria**
- [x] Change content for each location highlighted in the images below where it says income to "gross income" 
- [x] This change should be for ALL program years across PT & FT
- [x] Update this for the current year income exception in the application for students, partners, and on the change requests for both as well. See screen shots with highlighted locations below
- [x] Update "change" to "changed" in the partner current year income application. See second image bullet #1

**Technical ACs:**
- [x] All the ACs listed above are basically the technical ACs for this story.

### Student estimated income application
<img width="842" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOG9jQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0f1222eb49724e53c5376b7ab758a62914374e88/image.png" alt="image.png" />

### Partner estimated income application
<img width="712" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOHdjQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fe1847203f8a950f9c44759a308a10e6a1d174fa/image.png" alt="image.png" />

### Student estimated income change request
<img width="893" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODhjQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--32130690b529e8c882eed3cd8b754f5fb958620a/image.png" alt="image.png" />

### Partner estimated income change request
<img width="922" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjgyQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9c255b90cf2524a10acbc2f7f4eb3750138d254e/image.png" alt="image.png" />

### Estimate income for both students and partners (Banner and documents content)
- [x] Add a yellow warning banner directly above "Documents required" for the exceptions above (Student CY income, Partner CY Income) (banner not needed for change requests)
<img width="1055" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzhoQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--aa4765109d7935bcda9068ba48ff290a8f0c81ee/image.png" alt="image.png" />
- [x] Update the documents required section, starting with "Provide the following",  the student and partner estimate income question in the application (not needed for change requests)
> **Documents required:**

> • A letter explaining the circumstances that meet one or more of the Appeal Criteria (see above) and a detailed breakdown of how you calculated total estimated gross income (if applicable).

>**AND**

> • All relevant supporting documentation of the circumstance(s) (medical records, letter from physician, record of employment, layoff or strike, etc.).
> • (If the year matching your application number is still in progress): All relevant supporting documentation of estimated total gross income for the current year, including all year-to-date gross income (pay stubs, invoices, bank statements, Employment Insurance benefit statements, Record of Employment, etc.).
OR
> • (If the year matching your application number is complete): CRA Income Tax Return and gross income information for the year matching the student financial assistance application number.
<img width="756" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2tjQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e3de2bc70c7861c1e40304a8a6e5a589a40f8700/image.png" alt="image.png" />

### Additional Transportation

- [x] Add a yellow warning banner directly above "Documents required" for the additional transportation exception
<img width="1055" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzhoQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--aa4765109d7935bcda9068ba48ff290a8f0c81ee/image.png" alt="image.png" />
- [x] Update the additional transportation exception and change request required documentation content with the below content:
>**Documents Required:**

> • **A statement** describing how you calculated your weekly kilometers between your residence and school in relation to the following:
> - If it’s required for **clinical** or **practicum purposes**, please provide a **schedule and letter of explanation which includes the location.** 
> - If it’s required for **family related responsibilities** (e.g. driving children to and from school or daycare), please provide a **statement that outlines your circumstances.**   
> - If you are **unable to move closer to your school**, please provide a **statement that outlines your circumstances.**
AND

> •	**A copy of your vehicle insurance:**
> -	**Must** be insured for proper vehicle use (e.g. “to or from work or school a one-way distance in excess of XXkm…”)
> -	**Must** be listed as a driver of the vehicle or the registered owner.

<img width="714" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFVjQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--44c32bc8b197dbb5b53c31613537227e41afc158/image.png" alt="image.png" />

**Additional Context**
Feed back from ministry users that they were seeing consistent challenges around this wording and it was leading to a lot of repeat work on these exceptions

Was flagged as high criticality but I have concerns on this interpretation in comparison to other issues we have in our backlog. To be discussed in shower stall.
Template attached

[Business analysis request - Income Exception document requirements  (1).docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeWNMQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6fbd5dba73b1aee59a68d64c77229d03e8c54321/Business%20analysis%20request%20-%20Income%20Exception%20document%20requirements%20%20%281%29.docx)

**Comment by CarlyCotton** [comment]
@Joshua-Lakusta CY income exceptions one of the higher exception requests. Bumping up from 3.X to look at for FT release or sooner as  it's content only.

**Comment by CarlyCotton** [comment]
@Joshua-Lakusta to add screenshots and move to lather

**Comment by Joshua-Lakusta** [comment]
Moving back to get the soap as i am waiting on confirmation from business that this is the outcome they want.

**Comment by CarlyCotton** [comment]
Conversation in SOAK about increasing severity of content "Failure to provide the documents required will result in denial of the exception and may cause additional delays in application processing." into a banner.

**Comment by JerPearson** [comment]
"." added to two AC's per feedback from Lewis and Andew that they were missing.
